src/goto-programs/builtin_functions.cpp

```diff
@@ -29,6 +29,8 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include <linking/zero_initializer.h>
 
+#include <java_bytecode/java_types.h>
```

Comment:
Is this include acceptable (i.e. dependency from goto-programs onto java_bytecode), I think not but on the flip side, this code _is_ Java specific. (I guess the answer is, this function should not be in builtin_functions)